### PR TITLE
Remove unused codes from scim component

### PR DESCRIPTION
**Pull request by isharak**

### CODE REVIEW

File: components/scim/org.wso2.carbon.identity.scim.common/src/main/java/org/wso2/carbon/identity/scim/common/listener/SCIMUserOperationListener.java [hunk 1]
```diff
@@ -88,12 +68,6 @@ public boolean doPostAuthenticate(String userName, boolean authenticated,
                                       UserStoreManager userStoreManager)
             throws UserStoreException {
         try {
-            // If scim not enabled returns
-            if (!userStoreManager.isSCIMEnabled()) {
-                return true;
-            }
-
-
             String activeAttributeValue = userStoreManager.getUserClaimValue(userName, SCIMConstants.ACTIVE_URI, null);
```
Comment (darshanasbg) [inline components/scim/org.wso2.carbon.identity.scim.common/src/main/java/org/wso2/carbon/identity/scim/common/listener/SCIMUserOperationListener.java]: We need to remove the whole implementation of checking account lock scenario (in doPostAuthenticate method), since this is not related to SCIM and identity mgt component need to handle that.

**Comment by johannnallathamby** [comment]
Conflicts resolved and merged with #813
